sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -478,9 +475,10 @@ private void initializeResources() throws InterruptedException {
 
     private void declineOffers(SchedulerDriver driver, List<Protos.OfferID> acceptedOffers, List<Protos.Offer> offers) {
         final List<Protos.Offer> unusedOffers = OfferUtils.filterOutAcceptedOffers(offers, acceptedOffers);
+        LOGGER.info("Declining {} unused offers:", unusedOffers.size());
         unusedOffers.stream().forEach(offer -> {
             final Protos.OfferID offerId = offer.getId();
-            LOGGER.info("Declining offer: " + offerId.getValue());
+            LOGGER.info("  {}", offerId.getValue());
```

Comment:
OK, fair enough. My only concern here is that individual log lines become way more dependent on context for their meaning, which can make gleaning info from them a little bit more involved, but it's a matter of taste.